spec/models/local_authority_spec.rb

```diff
@@ -76,5 +76,13 @@
         expect(@local_authority.link_last_checked).to be_nil
       end
     end
+
+    describe "calculate_count_of_broken_links" do
```

Comment:
Small style niggle, I'm used to seeing method tests have a `#` in front of them, see http://betterspecs.org/#describe